Doc/library/os.rst

```diff
@@ -569,6 +569,38 @@ process and user.
       See the documentation for :func:`getgroups` for cases where it may not
       return the same group list set by calling setgroups().
 
+.. function:: setns(fd, nstype=0)
+
+   Reassociate the current with a namespace.
+   See the :manpage:`setns(2)` man page for more details.
+
+   If *fd* refers to a ``/proc/[pid]/ns/`` link, ``setns()`` reassociates the
```

Comment:
```suggestion
   If *fd* refers to a :file:`/proc/{pid}/ns/` link, ``setns()`` reassociates the
```

Use the appropriate role, which also gives us built-in explicit highlighting of variable portions.